AudreyW_Lab_1.ipynb

```diff
@@ -0,0 +1,598 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ax = sns.distplot(data_poisson3,\n",
+    "                  bins=30,\n",
+    "                  kde=True,\n",
+    "                  color='skyblue',\n",
+    "                  hist_kws={\"linewidth\": 15,'alpha':1})\n",
+    "ax.set(xlabel='Poisson Distribution', ylabel='Frequency')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Statistical question: What is the likelihood that I will be in (blank) location?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.8152632445237722\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Test run for the third Poisson graph\n",
+    "sigma_test5 = stats.poisson.cdf(4, mu=3,loc = 0)\n",
+    "print(sigma_test5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 68,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "prob_test5 = stats.poisson.pmf(sigma_test5, mu)\n",
+    "print(prob_test5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since the probability attained is very low it makes sense that sigma would then have to be very large as seen above."
+   ]
```

Comment:
Don't forget to answer questions D. and E. from this section of the lab after you've made your calculations!